docs/conf.py

```diff
@@ -362,7 +366,24 @@ def rstjinja(app, docname, source):
         source[0] = rendered
 
 
+def lol(app, docname, source):
+    """
+    lol
```

Comment:
I'm not going to object on the function name, but there needs to be a docstring that explains this all